.azure-pipelines-no-sgx.yml

```diff
@@ -51,7 +51,7 @@ jobs:
   container:
     image: ccfciteam/ccf-ci-18.04-oe-0.6:latest
     # --publish-all enables end-to-end tests to communicate over ports
-    options: --publish-all
+    options: --publish-all -v /var/log/ccf:/var/log/ccf
```

Comment:
Can you add a comment explaining what this option does? (everywhere its used)

Reply:
yes thanks